src/account/account.ts

```diff
@@ -166,10 +166,12 @@ export async function getAccountInformation(sender: string | SendTransactionFrom
     totalAssetsOptedIn: Number(account.totalAssetsOptedIn),
     totalCreatedApps: Number(account.totalCreatedApps),
     totalCreatedAssets: Number(account.totalCreatedAssets),
-    appsTotalExtraPages: account.appsTotalExtraPages ? Number(account.appsTotalExtraPages) : undefined,
-    rewardBase: account.rewardBase ? Number(account.rewardBase) : undefined,
-    totalBoxBytes: account.totalBoxBytes ? Number(account.totalBoxBytes) : undefined,
-    totalBoxes: account.totalBoxes ? Number(account.totalBoxes) : undefined,
+    appsTotalExtraPages: account.appsTotalExtraPages !== undefined ? Number(account.appsTotalExtraPages) : undefined,
```

Comment:
This stuff also should be added to AccountManager.getIfnormation

Reply:
Will add as `lastHeartbeatRound` and `lastProposedRound` and make `bigint` to match semantics of `validAsOfRound` on that type.